src/main/java/com/librato/disco/Level2CacheStrategy.java

```diff
@@ -0,0 +1,29 @@
+package com.librato.disco;
+
+import org.apache.curator.framework.recipes.cache.ChildData;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+public interface Level2CacheStrategy {
+    /**
+     * Signals to the strategy that these items were successfully found
+     * from a delegate cache.
+     */
+    void accept(List<ChildData> items);
```

Comment:
Would it make sense to also send the state of the current cache? Maybe just the count of entries in cache.